preload/resources/modulepreload-iframe.html

```diff
@@ -0,0 +1,20 @@
+<!DOCTYPE html>
+<body>
+    <script>
+        const m = new URL('module1.js', location.href).toString();
+        const observer = new PerformanceObserver(l => {
+            const entries = l.getEntriesByName(m);
+            if (entries.length === 1) {
+                import(m).then(() => {
```

Comment:
Do we have to import the module here after all? Don't we already have our results?

Reply:
We want to make sure the preloaded module is not just loaded but also consumed, meaning that importing it does not generate an additional RT entry.

Reply:
Ah sorry, I missed the part checking this in the parent document. Got it.